nullaway/src/main/java/com/uber/nullaway/handlers/contract/ContractUtils.java

```diff
@@ -0,0 +1,125 @@
+package com.uber.nullaway.handlers.contract;
+
+import static com.google.errorprone.BugCheckerInfo.buildDescriptionFromChecker;
+
+import com.google.errorprone.VisitorState;
+import com.sun.source.tree.Tree;
+import com.sun.tools.javac.code.Symbol;
+import com.uber.nullaway.ErrorMessage;
+import com.uber.nullaway.NullAway;
+import java.util.Map;
+import javax.annotation.Nullable;
+import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.element.AnnotationValue;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.ElementKind;
+import javax.lang.model.element.ExecutableElement;
+import javax.lang.model.element.TypeElement;
+
+/** An utility class for {@link ContractHandler} and {@link ContractCheckHandler}. */
+class ContractUtils {
+
+  /**
+   * Retrieve the string value inside an @Contract annotation without statically depending on the
+   * type.
+   *
+   * @param sym A method which has an @Contract annotation.
+   * @return The string value spec inside the annotation.
+   */
+  static @Nullable String getContractFromAnnotation(Symbol.MethodSymbol sym) {
+    for (AnnotationMirror annotation : sym.getAnnotationMirrors()) {
+      Element element = annotation.getAnnotationType().asElement();
+      assert element.getKind().equals(ElementKind.ANNOTATION_TYPE);
+      if (((TypeElement) element)
+          .getQualifiedName()
+          .contentEquals("org.jetbrains.annotations.Contract")) {
+        for (Map.Entry<? extends ExecutableElement, ? extends AnnotationValue> e :
+            annotation.getElementValues().entrySet()) {
+          if (e.getKey().getSimpleName().contentEquals("value")) {
+            String value = e.getValue().toString();
+            if (value.startsWith("\"") && value.endsWith("\"")) {
+              value = value.substring(1, value.length() - 1);
+            }
+            return value;
+          }
+        }
+      }
+    }
+    return null;
+  }
+
+  static void reportMatch(
+      Tree errorLocTree,
+      String message,
+      @Nullable NullAway analysis,
+      @Nullable VisitorState state) {
+    assert analysis != null && state != null;
```

Comment:
In various places in this class we have `@Nullable` arguments and then we immediately assert they are non-null.  Why not just get rid of the `@Nullable` and the `assert`?